test/queryClient.test.ts

```diff
@@ -32,4 +59,285 @@ describe('query client', () => {
+    });
+    assert(result, 'cannot request query GET_SERVICE_AGREEMENTS');
+    console.log(result.data.serviceAgreements.nodes)
+    expect(result.data.serviceAgreements).toBeTruthy()
+  }, 16000)
+
+  it('can query expired agreements', async () => {
+    const apolloClient = client.explorerClient;
+    const result = await apolloClient.query({
+      query: GET_EXPIRED_SERVICE_AGREEMENTS,
+      variables: {address:'0xCef192586b70e3Fc2FAD76Dd1D77983a30d38D04',now:date},
+    });
+    assert(result, 'cannot request query GET_EXPIRED_SERVICE_AGREEMENTS');
+    expect(result.data.serviceAgreements.nodes).toBeTruthy()
+  }, 16000)
+
+  it('can query project agreements', async () => {
+    const apolloClient = client.explorerClient;
+    const result = await apolloClient.query({
+      query: GET_SPECIFIC_SERVICE_AGREEMENTS,
+      variables: {deploymentId:'Qmdpka4MpaUtGP7B3AAoPji4H6X7a2ir53a1mxnUumqMm4',now:date},
+    });
+    assert(result, 'cannot request query GET_SPECIFIC_SERVICE_AGREEMENTS');
+    console.log(result.data.serviceAgreements.nodes)
+    expect(result.data.serviceAgreements.nodes).toBeTruthy()
+  }, 16000)
+
+  it('can query deployment by projectCid', async () => {
+    const apolloClient = client.explorerClient;
+    const result = await apolloClient.query({
+      query: GET_DEPLOYMENT,
+      variables: {deploymentId:'Qmdpka4MpaUtGP7B3AAoPji4H6X7a2ir53a1mxnUumqMm4'},
+    });
+    assert(result, 'cannot request query GET_DEPLOYMENT');
+    expect(result.data.deployment.id).toEqual('Qmdpka4MpaUtGP7B3AAoPji4H6X7a2ir53a1mxnUumqMm4')
+  }, 16000)
+
+  it('can query deploymnet indexer', async () => {
+    const apolloClient = client.explorerClient;
+    const result = await apolloClient.query({
+      query: GET_DEPLOYMENT_INDEXERS,
+      variables: {deploymentId:'Qmdpka4MpaUtGP7B3AAoPji4H6X7a2ir53a1mxnUumqMm4'},
+    });
+    assert(result, 'cannot request query GET_DEPLOYMENT_INDEXERS');
+    expect(result.data.deploymentIndexers.totalCount).toEqual(2)
+  }, 16000)
+
+  it('can query GET_DEPLOYMENT_INDEXERS_WITH_INDEXER', async () => {
+    const apolloClient = client.explorerClient;
+    const result = await apolloClient.query({
+      query: GET_DEPLOYMENT_INDEXERS_WITH_INDEXER,
+      variables: {indexerAddress:'0xf9e4E6307a3186991F153249294815228D3a4634'},
+    });
+    assert(result, 'cannot request query GET_DEPLOYMENT_INDEXERS_WITH_INDEXER');
+    expect(result.data.deploymentIndexers).toBeTruthy()
+  }, 16000)
+
+  it('can query get accepted offer', async () => {
+    const apolloClient = client.explorerClient;
+    const result = await apolloClient.query({
+      query: GET_ACCEPTED_OFFERS,
+      variables: {$address:'0xf9e4E6307a3186991F153249294815228D3a4634',$offerId:'1'},
+    });
+    assert(result, 'cannot request query GET_ACCEPTED_OFFERS');
+    console.log(result.data)
+    expect(result.data).toBeTruthy()
+    deepAssert(result.data.indexer);
+  }, 16000)
+
```

Comment:
need to be careful with testing with specific variables as if we deploy new contracts this data won't be there and we will have to update this.